pkg/cvo/sync_worker_test.go

```diff
@@ -113,16 +113,16 @@ func Test_statusWrapper_ReportGeneration(t *testing.T) {
 		next     SyncWorkerStatus
 		want     int64
 	}{{
-		previous: SyncWorkerStatus{Generation: 1, Step: "Apply", Done: 10, Total: 100},
-		next:     SyncWorkerStatus{Step: "RetreivePayload"},
-		want:     1,
+		previous: SyncWorkerStatus{Generation: 1, Done: 10, Total: 100},
+		//next:     SyncWorkerStatus{},
```

Comment:
nit: if we don't need `next`, probably just drop this line instead of commenting it out.